### PR TITLE
Adds records management handling.

### PR DESCRIPTION
**Pull request by nsbruce**
It is probably a good idea for @jeff-stacey to actually review this since if I buggered it up somewhere that has implications down the road.

### CODE REVIEW

File: lwatools/file_tools/handle_records.py [hunk 1]
```diff
@@ -4,127 +4,171 @@
...
-        if known_transmitter:
-            if known_transmitter not in known_transmitter_locations.keys():
-                raise ValueError(f"Unrecognized transmitter name {known_transmitter}. See utils/known_transmitters.py.")
+        if tx_type == 'tone' and known_transmitter not in known_transmitter_locations.keys():
+            raise ValueError(f"Unrecognized transmitter name {known_transmitter}. See utils/known_transmitters.py.")
 
         obs_dict = defaultdict() # handles keyerrors
         
+        if tbn_path.startswith('./'):
+            raise ValueError("Relative paths are not allowed")
```
Comment (jeff-stacey) [inline lwatools/file_tools/handle_records.py]: Why not? Can't we just use `Path.resolve()` to expand?

File: lwatools/file_tools/handle_records.py [hunk 2]
```diff
@@ -4,127 +4,171 @@
...
-        num_antennas = len(station_obj.antennas)
-        samples_per_frame = 512
-        num_samples = num_frames * samples_per_frame / num_antennas
-        duration = num_samples / tbnf.get_info('sample_rate')
-        obs_dict['duration'] = duration
-
-        # get size on disk
-        disk_size = os.path.getsize(tbn_path) # can also use tbnf.size
-        obs_dict['disk_size'] = disk_size
+        with LWASVDataFile(tbn_path) as tbnf:
```
Comment (jeff-stacey) [inline lwatools/file_tools/handle_records.py]: Did you mean to use LWADF here?

**Review by jeff-stacey — COMMENTED** [review]
looks good overall just a few comments

File: lwatools/file_tools/handle_records.py (hunk 2 above)
Comment (nsbruce) [reply lwatools/file_tools/handle_records.py]: yes

File: lwatools/file_tools/handle_records.py (hunk 1 above)
Comment (nsbruce) [reply lwatools/file_tools/handle_records.py]: Probably yea, I'll check

**Pull request merged**